sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/.gitignore

```diff
@@ -0,0 +1,362 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Most of these seem pretty generic and should already be covered in the repo's main gitignore - https://github.com/Azure/azure-sdk-for-net/blob/main/.gitignore

Can you check to see if this would cover everything here, or if there are SignalR specific entries here?

Reply:
The only thing not covered by main gitignore is "local.settings.json". Can "local.settings.json" be added to main gitignore? I think that might be helpful for all the function extension projects with function samples.